moped-editor/src/utils/activityLogHelpers.js

```diff
@@ -76,6 +76,7 @@ export const buildLookupQuery = tableNames => {
 export const getActivityLogTableNames = (response, lookupDataKey) => {
   const lookupTableData = response?.[lookupDataKey];
 
+  // would this length ever be more than one??
```

Comment:
I looked into this a little, and, even in a brand new project, this length is 1 and then grows as more events are added that need lookup tables. Maybe there was a time when it could be 0 and that changed over time?